e2e/playwright/tests/command/todo.spec.ts

```diff
@@ -0,0 +1,181 @@
+                await expect(todo.getDesc('openpr')).toHaveText('You have 1 open pull requests:');
+                await expect(todo.getDesc('assignments')).toHaveText('You have 19 assignments:');
+                await expect(todo.getDesc('reviewpr')).toHaveText('You have 1 pull requests awaiting your review:');
+                await expect(todo.getDesc('unread')).toHaveText('You have 1 unread messages:');
+
+                // * Assert the open pull request list has 1 items
+                const openPr = await todo.getList('openpr');
+                await expect(openPr.locator('li')).toHaveCount(1)
+
+                // * Assert the open pull request links are correct <REPO> <PR>
+                for (let i=0; i<1; i++) {
+                    await expect(openPr.locator('li').nth(i).locator('a').nth(0)).toHaveAttribute('href', repoRegex)
+                    await expect(openPr.locator('li').nth(i).locator('a').nth(1)).toHaveAttribute('href', prRegex)
+                }
+
+                // * Assert the review request list has 1 items
+                const reviewPr = await todo.getList('reviewpr');
+                await expect(reviewPr.locator('li')).toHaveCount(1)
+
+                // * Assert the pull request links are correct <REPO> <PR>
+                for (let i=0; i<1; i++) {
+                    await expect(reviewPr.locator('li').nth(i).locator('a').nth(0)).toHaveAttribute('href', repoRegex)
+                    await expect(reviewPr.locator('li').nth(i).locator('a').nth(1)).toHaveAttribute('href', prRegex)
+                }
+
+                // * Assert the assignments list has 1 items
+                const assignments = await todo.getList('assignments');
+                await expect(assignments.locator('li')).toHaveCount(19)
+
+                // * Assert the assignments links are correct <REPO> <ISSUE>
+                for (let i=0; i<1; i++) {
+                    await expect(assignments.locator('li').nth(i).locator('a').nth(0)).toHaveAttribute('href', repoRegex)
+                    await expect(assignments.locator('li').nth(i).locator('a').nth(1)).toHaveAttribute('href', issueRegex)
+                }
+
+                // * Assert the unread has 1 items
+                const unread = await todo.getList('unread');
+                await expect(unread.locator('li')).toHaveCount(1);
+
+                // # Refresh
+                await page.reload();
+
+                // * Assert that ephemeral has disappeared
+                await expect(page.locator(`#post_${postId}`)).toHaveCount(0);
+            });
+        });
+    },
+    unconnected: () => {
+        test.describe('/github todo command', () => {
+
+            test('from non connected account', async ({pages, page, pw}) => {
+                const {adminClient, adminUser} = await pw.getAdminClient();
+                const URL = await getGithubBotDM(adminClient, '', adminUser!.id);
```

Comment:
0/5 on this

```suggestion
                const URL = await getGithubBotDMPageURL(adminClient, '', adminUser!.id);
```

Reply:
Note that we are shadowing the `URL` class by doing this. I suggest we use `url` or something like `dmChannelURL`